src/application/deployer/index.go

```diff
@@ -67,6 +51,28 @@ func StartDeploy(deployConfig types.DeployConfig) error {
 	return deployApplication(deployConfig, imagesMap, prevTerraformFileContents)
 }
 
+func pushApplicationImages(deployConfig types.DeployConfig) (map[string]string, error) {
```

Comment:
Open to a different name here since we already have a `PushApplicationImages`

Reply:
Can we break "writeDockerCoposeFile" into its own function which returns the dockercompse dir? Thats most of the logic in this function